packages/opentelemetry-plugin-grpc-js/src/server/patchServer.ts

```diff
@@ -111,6 +121,26 @@ export function patchServer(
   };
 }
 
+/**
+ * Returns true if the server call should not be traced.
+ */
+function shouldNotTraceServerCall(
+  this: GrpcJsPlugin,
```

Comment:
can you please change `this` into `grpcJsPlugin`, I find it a bit misleading later in code

Reply:
If I change this to grpcJsPlugin then I cant access the `_config` of the plugin because it is a protected field. Do you have any suggestions around this?

Reply:
then function `shouldNotTraceServerCall` should not be a standalone function but a private of `GrpcJsPlugin `

Reply:
Because the function needs to be called in `patchServer(grpcJsPlugin)` I cant actually make the function a private member of the class. 

One alternative would be to remove the `this` keyword from `shouldNotTraceServerCall` and pass in the list of ignored gRPC methods in as parameter: 

```typescript
/**
 * Returns true if the server call should not be traced.
 */
function shouldNotTraceServerCall(
  metadata: grpcJs.Metadata,
  methodName: string,
  ignoreGrpcMethods?: IgnoreMatcher[]
): boolean {
  const parsedName = methodName.split('/');
  return (
    containsOtelMetadata(metadata) ||
    methodIsIgnored(
      parsedName[parsedName.length - 1] || methodName,
      ignoreGrpcMethods
    )
  );
}
```

Reply:
When the first parameter is named `this` in typescript it carries special meaning. It only carries typing info and is compiled away. Example [Playground Link](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABAUwB4EMC2AHANsgCigAsYBnALkXAGsw4B3MAGkXSrKgCcYwBzAJSIA3gChRAX1FA) to illustrate.

Reply:
Please make this function a private member of the class.

Reply:
Sorry my vs code was underlining the changes in red, but its all good. Sorry about the confusion! Will push this change

Reply:
Just checking in to see if there's any advice on this issue?

Reply:
Ah I see it is called from patchserver not from within the same class. You can make it a util function, but please do not use `call` to call it. It would be much better if you could pass any required configs or properties as arguments to the function.